typescript/cli/src/tests/warp/warp-rebalancer.e2e-test.ts

```diff
@@ -733,9 +733,7 @@ describe('hyperlane warp rebalancer e2e tests', async function () {
     );
 
     await startRebalancerAndExpectLog(
-      `Could not get rebalance quotes from ${chain3Metadata.name} to ${
-        chain2Metadata.name
-      }, for 5 token: All providers failed on chain unknown for method call and params`,
+      `Could not get rebalance quotes from ${chain3Metadata.name} to ${chain2Metadata.name}, for 5 token`,
```

Comment:
Needed to change this expected error, as the `All providers failed on chain unknown for method call and params` is now part of the error and stack trace and not captured in the chunks/lines.